pkg/templates/charts/toggle/hypershift/templates/hypershift-addon-manager-deployment.yaml

```diff
@@ -62,11 +62,13 @@ spec:
       hostPID: false
       hostIPC: false
       securityContext:
-        runAsNonRoot: true
+      #  runAsNonRoot: true
```

Comment:
if set runAsNonRoot to true, the hypershift pod cannot run in AKS cluster. but is it required for OCP cluster.
need discuss with hypeshfit-addon team about this .